src/common/hostPlatform.ts

```diff
@@ -75,6 +82,11 @@ abstract class UnixHostPlatform implements IHostPlatform {
     }
 
     public abstract getPlatformId(): HostPlatformId;
+
+    public getTargetPlatformCompatibility(targetPlatformName: string): boolean {
+        let targetPlatformId = TargetPlatformHelper.getTargetPlatformId(targetPlatformName);
```

Comment:
Is this fully correct? I think that this will attempt to allow iOS builds on linux which I believe will fail just as much as on windows.